src/Orleans/Telemetry/Consumers/ConsoleTelemetryConsumer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Orleans.Runtime
+{
+    public class ConsoleTelemetryConsumer : ITraceTelemetryConsumer, IExceptionTelemetryConsumer
```

Comment:
Today we output to console via `Trace` by installing a `ConsoleTraceListener`. This will change it to explicit output via `Console`. I wonder if there's any difference between these two paths in term or performance, locks, etc.